gateway/proxy_eth.go

```diff
@@ -200,6 +200,14 @@ func (gw *Node) EthGetBlockByNumber(ctx context.Context, blkNum string, fullTxIn
 	return gw.target.EthGetBlockByNumber(ctx, blkNum, fullTxInfo)
 }
 
+func (gw *Node) EthGetTransactionByBlockHashAndIndex(ctx context.Context, blkHash ethtypes.EthHash, txIndex ethtypes.EthUint64) (*ethtypes.EthTx, error) {
+	return gw.target.EthGetTransactionByBlockHashAndIndex(ctx, blkHash, txIndex)
+}
+
+func (gw *Node) EthGetTransactionByBlockNumberAndIndex(ctx context.Context, blkNum string, txIndex ethtypes.EthUint64) (*ethtypes.EthTx, error) {
+	return gw.target.EthGetTransactionByBlockNumberAndIndex(ctx, blkNum, txIndex)
```

Comment:
needs rate limiting and tipset lookback checking

see `EthGetBlockByNumber` in here for how to do this, it can mostly be copied from there